packages/botfuel-dialog/src/bot.js

```diff
@@ -134,7 +139,7 @@ class Bot {
       let botMessages: BotMessageJson[] = [];
       await this.middlewareManager.in(contextIn, async () => {
         logger.debug('handleMessage: responding');
-        botMessages = await this.respond(userMessage);
+        botMessages = await this.respond(userMessage, rawSentence);
```

Comment:
why can't we pass it in the message?

Reply:
@yangeorget actually we can't add it to the userMessage because it goes through the middlewares and let the possibility to modify the property and we don't want that. Also we tried to add a new property `raw` to the UserTextMessage but it's conflicting with Flow Types so it's very hard to do that this way.

One possibility could have been to add it to message options after the in middlewares are applied but there is no options in user messages